functional/pipeline.py

```diff
@@ -384,6 +385,19 @@ def cartesian(self, *iterables, **kwargs):
             transformations.cartesian_t(iterables, kwargs.get("repeat", 1))
         )
 
+    def remove_all(self, iters: Union[List, Tuple]) -> NoReturn:
```

Comment:
Instead of `Union[List,Tuple]`, could we use `Container` https://docs.python.org/3/library/collections.abc.html#collections.abc.Container

To remain consistent with the rest of the methods, should this method return the sequence itself (similar to the the drop method below) to keep chaining methods..

Reply:
under normal conditions， all of the method 'remove' should not have the return value.